### ISSUE TITLE
How to write templates with "$"?

### ISSUE THREAD

**Issue by pdog18** [xylo/intellij-postfix-templates#56]
* write
```
	ANY → Timber.d("$expr$ = \$$expr$$END$")
```

* get
```
        Timber.d("length = $expr$END$")
```

* want 
```
        Timber.d("length = $length")
```

**Comment by xylo** [comment]
Indeed. That looks like a bug. Have to check that.

**Comment by xylo** [comment]
Fixed in version 1.6.2.
